CONTRIBUTING.md

```diff
@@ -0,0 +1,21 @@
+# Contributing to this project
+
+## Running tests locally
+
+To run tests locally for a specific version of PHP:
+
+```bash
+$ rm -rf vendor composer.lock
+$ docker-compose up
```

Comment:
missing `-d`

Reply:
For me, It's not mandatory to run in detached mode.

Reply:
Not mandatory, but you'll have to open a new terminal to run the next command